src/FlipCounter.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+
+const FlipCounter = props => {
+  var mystring = String(props.title);
+  var arr = mystring.split("");
+  arr.reverse();
+
+  if (arr.length < 6) {
+    for (var i = 0; i < 6 - (arr.length - 4); i++) arr.push("0");
+  }
+  return (
+    <div>
+      <p>mystring</p>
+      <h1>
+        <table>
+          <tbody>
+            <tr>
+              <td id="main">{arr[5]}</td>
```

Comment:
This is code dublication. Imagine, if you have 100 numbers there. 

You can iterate through array to print values.